docs/source/en/_toctree.yml

```diff
@@ -78,8 +78,10 @@
     title: Multimodal
     isExpanded: false
   - sections:
+      - local: autoregressive_generation
```

Comment:
I think the whole section should be just behind natural language processing over audio and computer vision